log/log_test.go

```diff
@@ -0,0 +1,61 @@
+package log
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+)
+
+func TestNewFormatter(t *testing.T) {
```

Comment:
See the trouble with this is that it's ai generated :D you need to add some additional prompts.

You need to call NewFormatter with "" (any), and a "json", assert that the expected type is returned. (`_, ok := result.(*T) ; assert.True(t, ok)`). SLOC 4, coverage 100%.